src/moduls/mantenimientos/pages/exportPDF.vue

```diff
@@ -215,11 +215,11 @@ th {
 <script setup>
 import iconsSvg from '../../../components/iconsSvg.vue';
 import { fecha, info, area, nombres, contacto, documento, terminos, email, mensaje } from '../../../components/svg';
-import maintenance_apiHandler, { actions, maintenance_base_endpoint } from '../APIHandler.mantenimientos'
+import maintenance_apiHandler, { actions, customer_base_endpoint, maintenance_base_endpoint } from '../APIHandler.mantenimientos';
 import { ref, onMounted } from 'vue';
 import { useRoute } from 'vue-router';
-
-
+import {generateHTMLContent} from '../utilidades/arcPDF.JS'
```

Comment:
```suggestion
import {generateHTMLContent} from '../utilidades/arcPDF.js'
```

jaja El nombre da poca información si lo ves correcto podrías cambiarlo a algo más genérico o lo contrario
(opcional)